packages/corecount/corecount.1.0.0/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "1.2"
+maintainer: "Edgar Aroutiounian <edgar.factorial@gmail.com>"
+authors: "Edgar Aroutiounian <edgar.factorial@gmail.com>"
+homepage: "http://hyegar.com"
```

Comment:
Could you please change the homepage to the project homepage (e.g. `https://github.com/fxfactorial/ocaml-corecount`) rather than yours?